x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -3,26 +3,18 @@
 
 To use PKI in {es}, you configure a PKI realm, enable client authentication on
 the desired network layers (transport or http), and map the Distinguished Names
-(DNs) from the Subject field in the user certificates to roles. You create the mappings in a role mapping file or use the role mappings API.
+(DNs) from the Subject field in the user certificates to roles. You create the
+mappings in a role mapping file or use the role mappings API.
 
-If you want the same users to also be authenticated using certificates when they connect to {kib}, you must configure the {es} PKI realm to
-<<pki-realm-for-proxied-clients,allow delegation>> and to
-{kibana-ref}/kibana-authentication.html#pki-authentication[enable PKI authentication in {kib}].
-
-You can also use a combination of PKI and username/password authentication. For
+TIP: You can use a combination of PKI and username/password authentication. For
 example, you can enable SSL/TLS on the transport layer and define a PKI realm to
 require transport clients to authenticate with X.509 certificates, while still
-authenticating HTTP traffic using username and password credentials. You can
-also set `xpack.security.transport.ssl.client_authentication` to `optional` to
-allow clients without certificates to authenticate with other credentials.
-
-IMPORTANT: You must enable SSL/TLS with client authentication to use PKI when
-clients connect directly to {es}.
```

Comment:
> You can also set `xpack.security.transport.ssl.client_authentication` ... You must enable SSL/TLS with client authentication...

I moved this content into the "Enable client authentication on the desired network layers (transport or http)" step, since in my opinion it was out of context and therefore harder to understand in this introductory section.